src/args.jl

```diff
@@ -574,6 +575,7 @@ add_aliases(:xerror, :xerr, :xerrorbar)
 add_aliases(:yerror, :yerr, :yerrorbar, :err, :errorbar)
 add_aliases(:quiver, :velocity, :quiver2d, :gradient, :vectorfield)
 add_aliases(:normalize, :norm, :normed, :normalized)
+add_aliases(:show_empty_bins, :showemptybins, :showempty, :show_empty, :show_zeros, :showzeros)
```

Comment:
Do we need the `show_zeros` etc aliases? It seems like they might be appropriated for some other uses.

Reply:
I added these based on @daschw's suggested name for the attribute, but I don't mind removing them, or doing other name-bikesheddin